string currentRoom
bool exploration
string idObject
string objectRoom
geometry_msgs/Vector3 poseObject
uint16 pxObject
string[] objectsInFOV